GoogleDataTransport/GDTCCTLibrary/GDTCCTUploader.m

```diff
@@ -170,31 +158,45 @@ - (nullable NSURL *)serverURLForTarget:(GDTCORTarget)target {
                           p2[31], p1[32], p2[32], p1[33], p2[33], p1[34], p2[34], p1[35], '\0'};
     CSHServerURL = [NSURL URLWithString:[NSString stringWithUTF8String:URL]];
   });
+  static NSDictionary<NSNumber *, NSURL *> *uploadURLs;
+  static dispatch_once_t URLOnceToken;
+  dispatch_once(&URLOnceToken, ^{
+    uploadURLs = @{
+      @(kGDTCORTargetCCT) : CCTServerURL,
+      @(kGDTCORTargetFLL) : FLLServerURL,
+      @(kGDTCORTargetCSH) : CSHServerURL,
+      @(kGDTCORTargetINT) : [NSURL URLWithString:kINTServerURL]
+    };
+  });
+  return uploadURLs;
+}
 
++ (nullable NSURL *)serverURLForTarget:(GDTCORTarget)target {
 #if !NDEBUG
   if (_testServerURL) {
     return _testServerURL;
   }
 #endif  // !NDEBUG
 
-  switch (target) {
-    case kGDTCORTargetCCT:
-      return CCTServerURL;
-
-    case kGDTCORTargetFLL:
-      return FLLServerURL;
-
-    case kGDTCORTargetCSH:
-      return CSHServerURL;
+  return [self uploadURLs][@(target)];
```

Comment:
NIT: Can you fetch the URLs as a local dictionary and return the value from that dictionary?

Reply:
Fixed.